aiidalab/utils.py

```diff
@@ -151,9 +152,12 @@ def __init__(self, name, version):
         self.name = name
```

Comment:
Also per the comment below, I think it makes sense to canonicalize the name here, otherwise we'll keep hitting this issue.
```suggestion
        self.name = canonicalize_name(name)
```